.ddev/config.arm.yaml

```diff
@@ -1 +0,0 @@
-dbimage: massgov/mysql-sanitized:latest.arm
```

Comment:
Was this an intended change?

Reply:
Yes - it is unused. We use an env variable instead.